docs/ReleaseBranch.md

```diff
@@ -0,0 +1,38 @@
+### Creating a new Release Branch
+
+1. Create a branch name with the format `releases_v[version]` (for example: `releases_v87.0.0`).
```

Comment:
Maybe for consistency we should use the same versioning format for the name of the branch as in the versionName  from gradle(for example: `releases_v87.0` and not `releases_v87.0.0`)
@rvandermeulen What do you think?

Reply:
Ideally, we did want to have it as v87.0 as we do for AC, but we stuck to consistency with Fenix had started the v87.0.0 and _I think_ some automation is built around that format for now - so we need to check that nothing breaks there during release.

We should definitely switch them both to dropping the patch version though.